### PR TITLE
Feature/login screen

### PR DESCRIPTION
**Pull request by alandours**
- Add styles to login screen

- Add components:
  - Header
  - MainButton
  - SignButton

Screenshots:

### CODE REVIEW

File: src/navigators/AuthStack.js [hunk 1]
```diff
@@ -10,8 +10,8 @@ const Stack = createStackNavigator();
 
 const AuthStack = () => (
   <Stack.Navigator>
-    <Stack.Screen name={LOGIN_SCREEN} component={LoginScreen} />
-    <Stack.Screen name={SIGN_UP_SCREEN} component={SignUpScreen} />
+    <Stack.Screen name={LOGIN_SCREEN} component={LoginScreen} options={{ headerShown: false }} />
+    <Stack.Screen name={SIGN_UP_SCREEN} component={SignUpScreen} options={{ headerShown: false }} />
   </Stack.Navigator>
```
Comment (Stukz) [inline src/navigators/AuthStack.js:15]: we only have two screens here and they both hide the header, maybe you can use the navigation options on the `<Stack.Navigator/>` to apply to both of them

File: src/screens/LoginScreen/styles.js [hunk 2]
```diff
@@ -7,10 +7,19 @@ const styles = StyleSheet.create({
...
-    textAlign: 'center',
+  forgotPasswordButton: {
+    alignSelf: 'center',
+    color: '#000',
+    fontSize: 10.5,
+    marginBottom: 22,
+  },
+  facebookLoginButton: {
+    alignSelf: 'center',
+    color: '#000',
```
Comment (Stukz) [inline src/screens/LoginScreen/styles.js:18]: don't forget to use the **Colors** constants

File: src/screens/LoginScreen/index.js [hunk 3]
```diff
@@ -16,9 +20,15 @@ const LoginScreen = ({ navigation }) => {
...
     <View style={styles.container} testID={LOGIN_SCREEN}>
-      <Text style={styles.welcome}>{strings.SIGN_IN.title}</Text>
+      <Header />
       <LoginForm onSubmit={loginRequest} />
-      <Button testID="sign-up-button" title={strings.SIGN_UP.title} onPress={handleLogin} />
+      <TouchableOpacity testID="forgot-password-button">
+        <Text style={styles.forgotPasswordButton}>Forgot your password?</Text>
+      </TouchableOpacity>
+      <TouchableOpacity testID="facebook-login-button">
+        <Text style={styles.facebookLoginButton}>Connect With Facebook</Text>
```
Comment (Stukz) [inline src/screens/LoginScreen/index.js:29]: **[1]**

File: src/components/common/ErrorView/styles.js [hunk 4]
```diff
@@ -3,14 +3,14 @@ import { ERROR } from 'constants/colors';
...
-    padding: 12,
   },
   error: {
     color: ERROR,
-    fontSize: 14,
-    fontWeight: '500',
-    marginBottom: 8,
+    fontSize: 10,
+    lineHeight: 12,
+    marginTop: -16,
```
Comment (Stukz) [inline src/components/common/ErrorView/styles.js:13]: I'm just curious about this negative value, maybe could you add a little explanation to it

**Review by Stukz — COMMENTED** [review]
Hey @alandours, I just leave some minor comments to the PR, let me know if anything isn't clear. Also, don't forget to include some images/videos to display what the actual change looks like

File: src/components/common/ErrorView/styles.js (hunk 4 above)
Comment (alandours) [reply src/components/common/ErrorView/styles.js:13]: The Input components have a marginBottom to separate from each other and from the Sign In button. If there are any errors, ErrorView is displayed below that margin, making it look too far apart from the inputs. I added the negative marginTop so it looks like it does in [Figma](https://www.figma.com/file/5hBk8f8Q4NsAoh68WqShi4/Target-App)

**Comment by alandours** [comment]
Hi @Stukz, thanks for the review! I'll update the PR soon with your suggestions

**Review by guillermomachado — APPROVED** [review]
Looks good!

**Pull request merged**